app/src/main/res/values/strings.xml

```diff
@@ -396,6 +396,17 @@
     <string name="settingsAiEngineDescription">Select your AI writing assistant</string>
     <string name="settingsAppLock">App lock</string>
     <string name="settingsAuthorizeTracking">Authorize tracking</string>
+    <string name="settingsAutoAdvanceDescription">Select what to display when a message is archived or deleted\u0020</string>
+    <string name="settingsAutoAdvanceFollowingThreadDescription">Display following thread when archiving or deleting\n</string>
```

Comment:
There's a useless new line at the end of this translation. It also applies to other languages